cmsdb/campaigns/run2_2017_nano_uhh_v11/top.py

```diff
@@ -52,9 +52,9 @@
 #
 
 cpn.add_dataset(
-    name="ttz_llnunu_m10_amcatnlo",
+    name="ttz_zlep_m10toinf_amcatnlo",
```

Comment:
Just to clarify, this is named `zlep` beause it includes both charged leptons and neutrinos, right?

Reply:
Exactly. I just looked into the generator card to cross check: https://cms-pdmv-prod.web.cern.ch/mcm/requests?prepid=TOP-RunIISummer20UL16wmLHEGEN-00151&page=0&shown=140737488355327

I think we settled for "clep" to denote charged leptons.